zanata-war/src/main/java/org/zanata/email/RequestToJoinLanguageEmailStrategy.java

```diff
@@ -68,14 +66,12 @@ public String getSubject(Messages msgs) {
             InternetAddress[] toAddresses) {
         PersistentMap<String, Object> context = super.makeContext(
                 genericContext, toAddresses);
-        String safeHTML = HtmlUtil.SANITIZER.sanitize(htmlMessage);
         return context
                 .insert("fromLoginName", fromLoginName)
                 .insert("fromName", fromName)
                 .insert("replyEmail", replyEmail)
                 .insert("localeId", localeId)
                 .insert("localeNativeName", localeNativeName)
-                .insert("htmlMessage", safeHTML)
```

Comment:
I originally misunderstood the intention of this change because of my badly named variable.  

- [ ] However, I think we agreed that there is no need to discard "user message" as a feature.